src/shared-components/dropdown/desktopDropdown.tsx

```diff
@@ -93,22 +94,22 @@ export const DesktopDropdown = ({
           isOpen={isOpen}
           optionsContainerMaxHeight={optionsContainerMaxHeight}
           role="menu"
-          aria-activedescendant={value}
+          aria-activedescendant={`${value}`}
```

Comment:
I don't think this is quite what we want--if `value` is `undefined` or `null` we will then pass in `"undefined"` or `"null"`. 
```tsx
value ? `${value}` : undefined
```

 perhaps? (It looks like the DOM type is `string | undefined`).